Tasks/CopyPublishBuildArtifacts/CopyPublishBuildArtifacts.ps1

```diff
@@ -28,6 +28,13 @@ Write-Verbose "TargetPath = $TargetPath"
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
 
+$hostType = Get-TaskVariable $distributedTaskContext "system.hosttype"
+
+if($hostType -ine "build" )
```

Comment:
Not a mandatory change, but wouldn't `-ne 'release'` make more sense? If so then all messages would need to change as well.